force_gromacs/simulation_builders/base_gromacs_simulation_builder.py

```diff
@@ -51,28 +50,28 @@ class BaseGromacsSimulationBuilder(HasTraits):
     topology_data = Dict()
 
     #: Base folder for simulation data
-    _folder = Unicode()
+    _folder = Str()
```

Comment:
For these private attributes, is it worth implementing a special data structure that stores these `_*_file` attributes?

Reply:
Yes, that's probably a good idea

Reply:
https://github.com/force-h2020/force-bdss/pull/282

Reply:
I've had a go at implementing this in the `GromacsFileRegistry` class, it's relatively simple at the moment but it can probably be expanded upon if we start to support more file types